### ISSUE TITLE
Checkbox focus outline color is white and not accessible

### ISSUE THREAD

**Issue by jendowns** [carbon-design-system/ibm-security#517]
## Description

Related to https://github.com/carbon-design-system/carbon/issues/5984

The `Checkbox`'s `:focus` state uses the `$focus` color token, which is **white** for g90 and g100 themes.

I already propose a fix upstream, but I'd like to also get a quick fix here. Proposed solution is to override the focus outline color & use `$interactive-04` instead (which is [blue40 for dark themes](https://carbon-elements.netlify.app/themes/examples/preview/)).

### Steps for reproducing

Check out the `Checkbox` story & tab to focus on one of the boxes: https://ibm-security.carbondesignsystem.com/?path=/story/components-checkbox--checked
